### ISSUE TITLE
Disable more view popup?

### ISSUE THREAD

**Issue by Bxnnet** [nhn/tui.calendar#1302]
## Version
latest

## Test Environment
Ubuntu, newst TUI Version, Symfony

## Current Behavior

Since I can not find any option to customize the 'more view popup' i tried to disable it in order to create my own popup and therefore my idea was a bootstrap modal when clicking the 'x more' title. 
Works fine but I can not manage to disable the default popup, is there any possiblity to do so?

## Expected Behavior

A option to disable the more view popup same as for the other popups.

**Comment by adhrinae** [comment]
@Bxnnet
Unfortunately, we don't have an option to disable the default more view popup at this moment.

You can fork the repo and remove [this code](https://github.com/nhn/tui.calendar/blob/main/apps/calendar/src/components/dayGridMonth/gridCell.tsx#L198-L206)(and related code blocks) to disable the more view popup.

**Comment by Bxnnet** [comment]
@adhrinae Hey, thanks for the answer and solution idea.
Basically I dont necessarily need to disable it - that was just my idea to fix an issue: The more view popup gets rendered at a quite crazy location:
![Screenshot_1](https://user-images.githubusercontent.com/34416919/195253149-71734736-f0b3-4516-be3e-759ff7e4516d.png)

Here is the according html (after my backend calls finish, the calendar is rendered in die div id = calendar container.
![Screenshot_3](https://user-images.githubusercontent.com/34416919/195253361-66f61393-e750-45f4-82ed-d3bd1b9aed40.png)

**Comment by adhrinae** [comment]
Yeah, That's odd.

When the popup shows, the popup's position is affected by the browser's scroll position.

https://github.com/nhn/tui.calendar/blob/main/apps/calendar/src/components/dayGridMonth/gridCell.tsx#L124

Perhaps this calculation brings misbehavior to your application. I can only assume it from your information.

**Comment by Bxnnet** [comment]
Alright the problem seems to be that I render the calendar inside a bootstrap card deck. (Left side toolbar for calendar navigation, visibility, ... and right side the calendar view). As soon as I move the calendar container to an empty space (where it renders in full width), the popup location gets calculated correctly. 

Thanks for helping me finding the issue source but it should be possible, to use the calendar in my wanted way, shouldnt it?

Here is my full HTML in case you are willing to continue assisting me:
https://pastebin.com/KseNa5NL

**Comment by adhrinae** [comment]
> it should be possible, to use the calendar in my wanted way, shouldnt it?

I agree. We'll note it as a future requirement but please understand that it won't be added soon since the team behind TOAST UI is taking care of various projects(including internal projects).
